lib/grpc/client/adapter.ex

```diff
@@ -14,7 +14,7 @@ defmodule GRPC.Client.Adapter do
 
   @callback disconnect(channel :: Channel.t()) :: {:ok, Channel.t()} | {:error, any()}
 
-  @callback send_request(stream :: Stream.t(), contents :: binary(), opts :: keyword()) ::
```

Comment:
This is technically a breaking change, so let's hold off until we're ready for a new minor release

Reply:
Agreed :)